src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -247,7 +257,7 @@ def _extra_info_to_pgm_input_data(self, extra_info: ExtraInfo):  # pylint: disab
         nan = np.iinfo(dtype).min
         all_other_cols = ["i_n"]
         for component, data in self.pgm_output_data.items():
-            input_cols = power_grid_meta_data["input"][component].dtype.names
+            input_cols = power_grid_meta_data["input"][component]["dtype"].names
```

Comment:
was this a bug? or a deprecated feature of structured arrays? the original one feels more correct but i could be wrong